happi/cli.py

```diff
@@ -392,6 +426,389 @@ def transfer(ctx, name: str, target: str):
     transfer_container(client, item, target)
 
 
+benchmark_sort_keys = [
+    'name',
+    'avg_time',
+    'iterations',
+    'tot_time',
+    'max_time',
+    'import_time',
+]
+
+
+@happi_cli.command()
+@click.pass_context
+@click.option("-d", "--duration", type=float, default=0,
+              help="Specify how long in seconds to spend per device.")
+@click.option("-i", "--iterations", type=int, default=0,
```

Comment:
I was surprised to see `happi benchmark at2l0` do nothing; may be worthwhile to say "if duration/iterations unset, say 'please read --help"?

Reply:
And the running logic needs to be "run for at least this many iterations and for at least this total duration"